### DISCUSSION TITLE
2.3.30 Zeek or Suricata?

### DISCUSSION THREAD

**Discussion by darth-drew** [Security-Onion-Solutions/securityonion#3267] [Unsupported Versions] [answered]
In the new install, we obviously now have the option to use Suricata to generate metadata, which disables Zeek. I am curious what the communities consensus is on which is "better".

**Comment by weslambert** [comment, marked as answer]
For many, it's personal preference.  You might try spinning two different installs to see which you like best.  However, you may find that you get better performance with running a single IDS (like Suricata) on hardware with lesser specs, etc.
